File: Python Word examples/tables/add_table.py
import spirecloudword
from spirecloudword.api.tables_api import TablesApi
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.tables_api.TablesApi(configuration)

filename = "Template.docx"
node_path = "sections/0"
rows_count = 6
columns_count = 6
inputFolder = "input"
destFilePath = "output/Tables/add_table.docx"
api.add_table(filename, node_path, rows_count, columns_count, index_of_table=1, folder=inputFolder, dest_file_path=destFilePath)

File: JavaScript Word Examples/TablesApiDemo/Table/addTable.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TablesApi(apiClient);
    
    var filename = "Table.docx";
    var nodePath = "Section/0/Body/0";
    var rowsCount = 6;
    var columnsCount = 6;
    var destFilePath = "output/Tables/AddTable_1.docx";
    var opts = {
        storage: "",
        folder: 'Tables',
        indexOfTable: 0,
        password: null
    };
    instance.addTable(filename, nodePath, rowsCount, columnsCount, destFilePath, opts,
    function (error, data, response) {
         if (error) throw error;
    });
})();